docs/content/docs/guide/running.md

```diff
@@ -18,6 +18,7 @@ with that PipelineRun, and it will run if the following conditions are met:
   - The author is the owner of the repository.
   - The author is a collaborator on the repository.
   - The author is a public member on the organization of the repository.
+  - The author has permissions to push to branches inside the repository.
```

Comment:
@chmouel do you think i need to mention GitHub here because this is possible today only for GitHub
As this sentence shows that support for all provider isn't it

Reply:
That's the thing I am not sure about,

The issue you fixed only shows up with Bots with Github Apps generated token (I believe but we need to double check)

I don't think that can happen with webhook. And anyway if the user can push to a branch, the user would be collaborator and then allowed to push.

So feel free to keep it like this.

Reply:
If perticular repository has webhook configuration and dependabot is configured for that repository even webhook can work here

Reply:
i am not talking specifically about github webhook but all other git providers as well which uses webhoo mechanism

Reply:
Ok got it :+1: